server/server.go

```diff
@@ -133,6 +135,35 @@ func (s *Server) isUnixSocket() bool {
 	return s.cfg.Socket != ""
 }
 
+func (s *Server) forwardUnixSocketToTCP(socket string, addr string) {
+	if sock, err := net.Listen("unix", socket); err == nil {
```

Comment:
We should store the socket listener in `Server` and close it in in `Close` method if both TCP and socket is enabled.